#-------------------------
#-------------------------
#-------------------------

#The try block lets you test a block of code for errors.

#The except block lets you handle the error.

#The else block lets you execute code when there is no error.

#The finally block lets you execute code, regardless of the result of the try- and except blocks.

#-------------------------
#-------------------------
#-------------------------

#-------------------------
#-------------------------
#-------------------------

#try:
#	value = 2
#	del value
#	print(value)
#except NameError:
#	print("NameError: Variable value is not defined")
#except:
#	print("Other Error: Variable value is not defined")
#finally:
#	print("Hello, World!")

#-------------------------

#try:
#	raise Exception("This is an exception!")
##	raise TypeError("This is an exception!")
#except NameError:
#	print("NameError: Variable value is not defined")
#except TypeError:
#	print("TypeError: Variable value is not defined")
#except:
#	print("Other Error: Variable value is not defined")
#finally:
#	print("Hello, World!")

#-------------------------
#-------------------------
#-------------------------
